include/benchmark/benchmark_api.h

```diff
@@ -414,16 +416,18 @@ class State {
   // Range arguments for this run. CHECKs if the argument has been set.
   BENCHMARK_ALWAYS_INLINE
   int range_x() const {
```

Comment:
I've removed range_x() and range_y().
Also, ArgPair() and RangePair() have been removed.